content/resources/sta256.md

```diff
@@ -0,0 +1,75 @@
+---
+title: STA256
+icon: /icons/sta256.svg
+desc: Probability and Statistics I
+link: sta256
+tags:
+  - utm-math
+---
+
+# Probability and Statistics I
+
+## Prerequisite:
+
+MAT134H5 or MAT136H5 or MAT134Y5 or MAT135Y5 or MAT137Y5 or MAT157Y5 or 75%+ in
+MAT133Y5
+
+## What is it about?
+
+STA256 is a probability course that goes into details of how probability works.
+It teaches you to analyse data. For CS students it is a must course to get into
+started with ML. It is also a prerequisite for CS Major and Specialist.
+
+## Topics Covered
+
+- Basic Probability
+- Random Variable
+  - Continuous random variable
+    - Probability Mass Function
+    - Cumulative Distribution Function
+      - Probability distributions
+      - Bernoulli Distribution
+      - Poisson Distribution
+      - Negative Binomial Distribution
+  - Discrete random variables
+    - Probability Density Function
+    - Cumulative Distribution Function
+      - Probability distributions
+      - Gamma Distribution
+      - Normal Distribution
+- Vectors
+- Expectation
+
+## Resources
+
+<grid-1-x-2
+title="Intro to Statistics"
+link='https://www.youtube.com/watch?v=LMSyiAJm99g'
+img-Src="https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Ftse2.mm.bing.net%2Fth%3Fid%3DOIP.yqZtQlSmefkNp86Ww6pIVgHaHa%26pid%3DApi&f=1"
+button="Start Listening!"
+desc="This is an introduction to statistics. Let's you know what is statistics."></grid-1-x-2>
+
+<grid-1-x-2
+:reversed=true
+title="Bernoulli Distribution"
+link='https://www.youtube.com/watch?v=nl9WiZMZnYs'
+img-Src="https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Ftse3.mm.bing.net%2Fth%3Fid%3DOIP.l-dxGo8AUkD0-IjpStkAqgHaEK%26pid%3DApi&f=1"
+button="Check it out!"
+desc="This is an introduction to Bernoulli Distribution a very important distribution that you will learn during the course."></grid-1-x-2>
+
+<grid-1-x-2
+title="Random Variable"
+link='https://www.youtube.com/watch?v=dOr0NKyD31Q'
+img-Src="https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Ftse2.mm.bing.net%2Fth%3Fid%3DOIP.2Soqm2ofSDzcfWEnp1DM8gHaFj%26pid%3DApi&f=1"
+button="Start Listening!"
+desc="This is an introduction to Random Variable a very important part of the course."></grid-1-x-2>
+
+## What courses are avaliable after STA256:
+
+### CS Major/Specialist Graduation:
```

Comment:
This can potentially also be moved to another page (Just like the POSt one that I suggested above)